release/build_release.sh

```diff
@@ -109,11 +109,11 @@ function create_branch {
 		original_branch_name="${LIFERAY_RELEASE_GIT_PREV_REF}"
 	fi
 
-	local last_commit_sha=$(curl -s -H "Authorization: token ${LIFERAY_RELEASE_GIT_TOKEN}" https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/"${original_branch_name}" | jq -r '.object.sha')
+	local last_commit_sha=$(curl -s -H "Authorization: token ${LIFERAY_RELEASE_GITHUB_TOKEN}" https://api.github.com/repos/liferay/liferay-portal-ee/git/refs/heads/"${original_branch_name}" | jq -r '.object.sha')
```

Comment:
@kiwm, can you please move this renaming to the first commit?

This commit should have only the disabling of the function `create_branch` :wink: